spec/std/xml/xml_spec.cr

```diff
@@ -168,45 +168,205 @@ describe XML do
     errors[0].to_s.should eq("Opening and ending tag mismatch: people line 1 and foo")
   end
 
-  it "gets root namespaces scopes" do
-    doc = XML.parse(<<-XML
+  describe "#namespace" do
+    describe "when the node has a namespace" do
+      describe "with a prefix" do
+        it "return the prefixed namespace" do
+          doc = XML.parse(<<-XML
+          <?xml version="1.0" encoding="UTF-8"?>
+          <openSearch:feed xmlns:foo="http://www.w3.org/2005/Atom" xmlns:openSearch="http://a9.com/-/spec/opensearchrss/1.0/"></feed>
+          XML
+          )
+
+          namespace = doc.root.not_nil!.namespace.should_not be_nil
```

Comment:
Technically, there's no difference for a `Namespace | Nil` type, but I think the expressiveness of a positive expectation is really helpful. Especially since that type is used in the following lines.
```suggestion
          namespace = doc.root.not_nil!.namespace.should_be a(XML::Namespace)
```